doc/run.txt

```diff
@@ -1,13 +1,14 @@
 Synopsis
 
-    cordova-cli run [BUILD] [PROD] [TARGET] [EXP] [PLATS] [-- POPTS]
+    cordova-cli run [BUILD] [PROD] [TARGET] [EXP] [PLATS] [LIST] [-- POPTS]
```

Comment:
Actually, LIST is probably exclusive to BUILD,

so:

Replace [BUILD] with [MODE], and change BUILD to MODE: --list|--nobuild